Assets/Scripts/Game/UserInterfaceWindows/DaggerfallTravelMapWindow.cs

```diff
@@ -1231,9 +1231,33 @@ bool FindLocation(string name)
             string[] locations = currentDFRegion.MapNames.OrderBy(p => p).ToArray();
             name = name.ToLower();
 
+            // this next code block addresses a bug reported by user "electrorobobody" on forums 
+            // the bug prevented one to find locations with their names being a substring of
+            // a different location, e.g. in province Daggerfall "Kingsly Manor" could not be
+            // found via search because there exists a "Buckingsly Manor" that was returned instead
```

Comment:
I don't think we need this paragraph. And the bug reporter may not necessarily want their name put in the source code.